src/pages/Sketch/components/Canvas.js

```diff
@@ -40,58 +41,58 @@ function Canvas() {
     const [isDrawing, setIsDrawing] = useState(false);
     const [type, setType] = useState("pen");
     const [typeState, setTypeState] = useState(null);
-    const [downPoint, setDownPoint] = useState({x: "", y: ""});
-    const [mousePosition, setMousePosition] = useState({x: "0", y: "0"});
+    const [downPoint, setDownPoint] = useState({ x: "", y: "" });
+    const [mousePosition, setMousePosition] = useState({ x: "0", y: "0" });
 
-    const [canvasWidth, setCanvasWidth] = useState(window.innerWidth-50);
-    const [canvasHeight, setCanvasHeight] = useState(window.innerHeight-100);
+    const [canvasWidth, setCanvasWidth] = useState(window.innerWidth - 50);
+    const [canvasHeight, setCanvasHeight] = useState(window.innerHeight - 100);
 
     useEffect(() => {
         window.addEventListener('resize', () => {
-            setCanvasWidth(window.innerWidth-50);
-            setCanvasHeight(window.innerHeight-100);
+            setCanvasWidth(window.innerWidth - 50);
+            setCanvasHeight(window.innerHeight - 100);
         })
 
         return () => {
-            window.removeEventListener('resize', () => {});
+            window.removeEventListener('resize', () => { });
```

Comment:
```suggestion
            window.removeEventListener('resize', handleResizeListener);
```
This is not the correct way to remove the event listeners. You'll have to create `handleResizeListener` (or name any) function outside of useEffect scope and this will be a callback function that you've to pass while adding an event listener (i.e, in line number 51) as well as while removing the event listener.

Reply:
Sir, these are already existed code , done by other contributers.

Anyway thanks for your suggestions, I will make the change.

Reply:
@nijin26 I understand but better to make it look more readable and it's always a good practice to update the previously created code and refactor it in a way that will make it more efficient. 

P.S Just a suggestion: You have to make certain changes in the codebase (at least the files you're working on) irrespective of whether you've worked on it or others might have worked on that block of code, just to handle certain things in right way. 🙂

Reply:
Yes Sir,
I will do that .😇